recipes-kernel/linux/files/0024-net-phy-dp83867-Add-binding-for-the-status-led-confi.patch

```diff
@@ -0,0 +1,154 @@
+From 85495623310031d4ec0eeedacd1b0f45d5830100 Mon Sep 17 00:00:00 2001
+From: zengchao <chao.zeng@siemens.com>
+Date: Wed, 6 Nov 2019 11:21:49 +0800
+Subject: [PATCH 24/24] net:phy:dp83867:Add binding for the status led
+ configuration option
+
+the DP83867 has different function option for the status led.
+It is possible to set the status led for different function.
+
+refer datasheet:
+http://www.ti.com/product/DP83867IR/datasheet
+
+Signed-off-by: Chao Zeng <chao.zeng@siemens.com>
+---
+ .../devicetree/bindings/net/ti,dp83867.txt    |  5 ++++
+ .../boot/dts/ti/k3-am65-iot2050-pg2.dtsi      | 10 +++++++
+ drivers/net/phy/dp83867.c                     | 27 ++++++++++++++++++-
+ include/dt-bindings/net/ti-dp83867.h          | 18 +++++++++++++
+ 4 files changed, 59 insertions(+), 1 deletion(-)
+
+diff --git a/Documentation/devicetree/bindings/net/ti,dp83867.txt b/Documentation/devicetree/bindings/net/ti,dp83867.txt
+index 388ff48f53aee..69b6b0d74c563 100644
+--- a/Documentation/devicetree/bindings/net/ti,dp83867.txt
++++ b/Documentation/devicetree/bindings/net/ti,dp83867.txt
+@@ -42,6 +42,11 @@ Optional property:
+ 				    Some MACs work with differential SGMII clock.
+ 				    See data manual for details.
+ 
++	- ti,led-sel	- This configure the status led. See dt-bindings/net/ti-dp83867.h
++					  for different status led settings,select different configurations
++
++	- ti,led-name   - Correspond name of led source.
++
+ Note: ti,min-output-impedance and ti,max-output-impedance are mutually
+       exclusive. When both properties are present ti,max-output-impedance
+       takes precedence.
```

Comment:
Bindings need to be submitted separately - and then approved by the device tree community. Could you prepare such a series and submit that soon? We need the feedback to continue this.

Reply:
I just implement this binding. you mean need to separate the binding with dp83867 driver?
 I would follow the community rule to send a pr to upstream kernel

Reply:
I only meant separating the binding patch from the implementation patch.